FEM/invariants.h

```diff
@@ -0,0 +1,131 @@
+#ifndef INVARIANTS_H
+#define INVARIANTS_H
+
+#include <cfloat>
+// FEM-Makros
+#include "makros.h"
+#include "tools.h"
+
+#include "Eigen/Dense"
```

Comment:
move this right after line 4. The include order is
1. c++ standard lib
2. system lib
3. third party lib
4. OGS lib